crates/utils/src/settings.rs

```diff
@@ -56,7 +97,16 @@ pub struct EmailConfig {
 #[derive(Debug, Deserialize, Clone)]
 pub struct CaptchaConfig {
   pub enabled: bool,
-  pub difficulty: String, // easy, medium, or hard
+  pub difficulty: String,
```

Comment:
Can we turn this into an enum?

Reply:
I doubt it, since json / hjson doesn't support enums.

The API code does this:

```
    let captcha = match captcha_settings.difficulty.as_str() {
      "easy" => gen(Difficulty::Easy),
      "medium" => gen(Difficulty::Medium),
      "hard" => gen(Difficulty::Hard),
      _ => gen(Difficulty::Medium),
    };
```

Reply:
I mean in Rust, so that it gets serialized to string automatically.